mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -326,12 +323,12 @@ func TestAccDNSRecordSet_changeRouting(t *testing.T) {
 				ImportStateVerify: true,
 			},
 			{
-				Config: testAccDnsRecordSet_routingPolicyGEO(networkName, backendName, forwardingRuleName, zoneName, 300),
+				Config: testAccDnsRecordSet_routingPolicy(zoneName, 300),
 			},
 			{
-				ResourceName:  "google_dns_record_set.foobar",
-				ImportStateId: fmt.Sprintf("%s/%s/test-record.%s.hashicorptest.com./A", getTestProjectFromEnv(), zoneName, zoneName),
-				ImportState:   true,
+				ResourceName:      "google_dns_record_set.foobar",
+				ImportStateId:     fmt.Sprintf("%s/%s/test-record.%s.hashicorptest.com./A", getTestProjectFromEnv(), zoneName, zoneName),
+				ImportState:       true,
```

Comment:
Weird indentations.

Reply:
I guess I was trying to align with `ImportStateVerify: true,`